packages/react-tracking-pixel/src/tests/TrackingPixel.test.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import {mount} from 'enzyme';
+import Preconnect from '@shopify/react-preconnect';
+
+import TrackingPixel from '../TrackingPixel';
+
+describe('<TrackingPixel />', () => {
+  const url = '/my/url/to/somewhere';
+
+  it('renders a very tiny iframe', () => {
+    const trackingPixel = mount(<TrackingPixel url={url} />);
+    expect(trackingPixel.find('iframe').props()).toMatchObject({
+      sandbox: 'allow-scripts',
+      scrolling: 'no',
+      frameBorder: 0,
+      height: 1,
+      width: 1,
+    });
```

Comment:
Verify that `style` contains `display: none`.

Reply:
Added an expect statement to check the property is present